pytradfri/device.py

```diff
@@ -48,8 +48,7 @@ def reachable(self):
 
     @property
     def has_light_control(self):
-        return (ATTR_LIGHT_CONTROL in self.raw and
-                len(self.raw.get(ATTR_LIGHT_CONTROL)) > 0)
+        return (self.raw is not None and len(self.raw.get(ATTR_LIGHT_CONTROL,"")) > 0)
```

Comment:
Travis returned these errors:

```The command "py.test" exited with 0.
1.60s$ flake8
./pytradfri/device.py:51:77: E231 missing whitespace after ','
./pytradfri/device.py:51:80: E501 line too long (86 > 79 charact```

Reply:
Which is interpreted as you need to break the line into two as a row many not exceed 79 chars and you need to write ```(ATTR_LIGHT_CONTROL, "")```

Reply:
When I started doing this, I quickly realized the power of installing pylint and then running:
```pylint device.py```

It will list the syntax errors (if any) in your code. Travis runs the same checks and will not green flag the change until all tests have pasts. Green flags is, in turn, needed before a merge of the PR is done.